packages/components/src/dialogs/ActionButtonDialogTrigger.tsx

```diff
@@ -1,15 +1,18 @@
+/* eslint-disable camelcase */
```

Comment:
Unnecessary. In cases where we do need to go against eslint rules, should opt for disabling a single line only wherever possible.
```suggestion
```